### DISCUSSION TITLE
Is MEV-proxy on ETH supported?

### DISCUSSION THREAD

**Discussion by konstiantyn** [coinfoundry/miningcore#1113] [Ideas]
Is it possible on ETH pool?
https://docs.flashbots.net/flashbots-auction/miners/advanced/interacting-with-relay
https://github.com/flashbots/mev-proxy

**Comment by ei8ht187** [comment]
It is possible. 2miners.com offers payout of MEV: https://2miners.com/eth-mining-pool
